pkg/template/api/types.go

```diff
@@ -13,6 +13,9 @@ type Template struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Optional: An instructional message for the user processing this template (escaped parameters will be substitued).
```

Comment:
Changing to "An instructional message for the user processing this template (parameter substitution will be performed)." for clarity.